examples/standard/app-armor/app-armor.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: starlingx.windriver.com/v1
```

Comment:
Same comment as the aiosx example. Need a patch to apply a new example with the basic fields

Reply:
Added the required configuration